ee/models/license.py

```diff
@@ -78,6 +78,7 @@ class License(models.Model):
         AvailableFeature.SAML,
         AvailableFeature.SSO_ENFORCEMENT,
         AvailableFeature.ROLE_BASED_ACCESS,
+        AvailableFeature.DATA_COLOR_THEMES,
```

Comment:
This isn't used so you don't need to add here.

Reply:
Removed. This brings up another question though: Are we ok with soft-gating features like this? You could still use the API to create custom themes and then use them, we just don't show the UI for it.

Also, should I add special handling for users who downgrade? They could have a custom theme in place and then there's no way to go back to the default theme.

Reply:
We can also add these on the server, available features is stored on the organization so no paygate but if you search the API you'll see some examples.

Reply:
Cool, I'll:
(a) shadow the default theme id field on the team serializer to always return the default theme if the feature isn't available and
(b) add a feature check on the create/update endpoints for themes